spec/lib/video_info/providers/youtube_playlist_spec.rb

```diff
@@ -3,6 +3,7 @@
 [nil, 'AIzaSyA6PYwSr1EnLFUFy1cZDk3Ifb0rxeJaeZ0'].each do |api_key|
   describe VideoInfo::Providers::YoutubePlaylist do
     before(:each) do
+      #TODO: I think this is unnecessary.  Should be removed?
```

Comment:
@thibaudgg What do you think of this?  I'm pretty sure since we pass in the api key we don't need to set it.  Since the tests dont run in my repo tho didn't want to remove this since I can't reliable run the whole test suite.

Reply:
Yeah that's weird, or the `[nil, 'AIzaSyA6PYwSr1EnLFUFy1cZDk3Ifb0rxeJaeZ0'].each do |api_key|` loop could be removed as well, doesn't make sense to test twice the same thing. 🤔

Reply:
Let me know what you'd like! If possible hoping to get this merged sometime soon as I'm hoping to use it for another project.  Really appreciate your fast reply and helping me through these changes thus far!

Reply:
I would remove `[nil, 'AIzaSyA6PYwSr1EnLFUFy1cZDk3Ifb0rxeJaeZ0'].each do |api_key|`, no reason to test two times the same things. 👍

Reply:
@thibaudgg removed!